### DISCUSSION TITLE
[ Shaders ] Aali .post file

### DISCUSSION THREAD

**Discussion by DLPB2** [julianxhokaxhiu/FFNx#169] [Q&A]
Allowed for impressive post effect - I used one and tweaked with The Reunion.  This now seems to be discontinued.

See post.psx 

https://drive.google.com/file/d/1YgZamUfpPdo_KiUxuB21raDevN2l4gWm/view?usp=sharing

**Comment by julianxhokaxhiu** [comment]
You can build your own post shader effects at your own pleasure, by compiling it with `shaderc` from bgfx ( feel free to check the vendor folder for that ) and using this as a basis: 
- Vert: https://github.com/julianxhokaxhiu/FFNx/blob/master/misc/FFNx.post.vert
- Frag: https://github.com/julianxhokaxhiu/FFNx/blob/master/misc/FFNx.post.frag

Cheers!

**Comment by DLPB2** [comment]
Tried to compile using your examples and it fails for both. Vert mentions an error with #include on line 70.  Is there a proper compile tutorial for this somewhere? How did you compile them?  Perhaps it's the shaderc build.  Do you have a good binary?

**Comment by julianxhokaxhiu** [comment]
There you go: https://github.com/julianxhokaxhiu/FFNx/blob/master/src/CMakeLists.txt#L49
